frontend/src/common/helpers/storageHelper.ts

```diff
@@ -1 +1,7 @@
-export const getAccessToken = () => localStorage.getItem('token');
+const tokenFieldName = 'token';
```

Comment:
Better to create enum **localStorageFields** with fields like Token = 'token' and then use it for methods.